landingai/visualize.py

```diff
@@ -26,35 +24,47 @@ def overlay_predictions(
     types = {type(pred) for pred in predictions}
     assert len(types) == 1, f"Expecting only one type of prediction, got {types}"
     overlay_func = _OVERLAY_FUNC_MAP[types.pop()]
-    return overlay_func(predictions, image)
+    return overlay_func(predictions, image, options)
 
 
 def overlay_bboxes(
-    predictions: list[ObjectDetectionPrediction], image: np.ndarray
+    predictions: list[ObjectDetectionPrediction],
+    image: np.ndarray,
+    options: dict[str, Any] = None,
 ) -> Image.Image:
     "Draw bounding boxes on the input image and return the image with bounding boxes drawn."
-    color = (255, 0, 0)
-    thickness = 2
+    import bbox_visualizer as bbv
+
+    if options is None:
+        options = {}
+    label_type = options.get("label_type", "default")
     for pred in predictions:
```

Comment:
What if the user accidently pass non-OD detections?? How will this fail? Can we detect and return an error?